egs/ljspeech/tts2/run.sh

```diff
@@ -182,13 +182,13 @@ if [ ${stage} -le 4 ] && [ ${stop_stage} -ge 4 ]; then
            --resume ${resume} \
            --train-json ${tr_json} \
            --valid-json ${dt_json} \
-           --config ${decode_config}
+           --config ${train_config}
 fi
 
 if [ ${n_average} -gt 0 ]; then
     model=model.last${n_average}.avg.best
 fi
-outdir=${expdir}/outputs_${model}_$(basename ${decode_config%.*})
+outdir=${expdir}/outputs_${model}_$(basename ${train_config%.*})
```

Comment:
`outdir` should be derived from `decode_config`.
`expdir` comes from `train_config`.

```suggestion
outdir=${expdir}/outputs_${model}_$(basename ${decode_config%.*})
```